OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -1,31 +1,23 @@
 # Default values for iqserver.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
+replicaCount: 1
+image:
+  repository: sonatype/nexus-iq-server
+  pullPolicy: IfNotPresent
+  
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
-  imagePullPolicy: IfNotPresent
-  imagePullSecret: ""
+  hostname: iq-server.demo
+  imageName: sonatype/nexus-iq-server
+  imageTag: 1.90.0
```

Comment:
is there a requirement that imageTag and imageName be separate for something we're doing?  The operator framework compels us to combine these into just imageName, so could we do that here too?

Reply:
it makes it easier to override on the command line.